meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-bsp/uefi/files/0008-enable-rootfs-redundancy.patch

```diff
@@ -10,4 +10,4 @@ index 2c619e21..b254ae76 100644
 +								data = [01 00 00 00];
```

Comment:
Similar question about https://github.com/OE4T/meta-tegra/commit/d421bf5cfe1c3ad876984d12a4a6c7158d67d6b7 - do we still need this change?

Reply:
You are right, I think both of these can be removed now that we have removed the local custom flash xml files, will test

Reply:
Removed this, and it still works.